packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -261,18 +260,27 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
 
   // Sync loading state into the layout.
   layout.isLoading = isLoading;
+
+  let focusedKey = state.selectionManager.focusedKey;
+  let focusedItem = gridCollection.getItem(state.selectionManager.focusedKey);
+  if (focusedItem?.parentKey != null) {
+    focusedKey = focusedItem.parentKey;
+  }
```

Comment:
Do we need this logic still or can we just do `focusedKey = state.selectionManager.focusedKey;`? With the row focus logic change, we hopefully won't ever have focusedKey be the grid cell key

Reply:
good catch, updating this